### ISSUE TITLE
Help Regarding the code

### ISSUE THREAD

**Issue by yashrooprai** [CPSAHWAL/RapidKen.AI-Test#1]
Hey, I am also working on a similar project in which I need to user BERT to extract keyphrases, can you help me out?

**Comment by CPSAHWAL** [comment]
Hey, how can I help you? You may check the code of [Pranav's github](https://github.com/pranav-ust/BERT-keyphrase-extraction) I referred for this.

**Comment by CPSAHWAL** [comment]
https://github.com/pranav-ust/BERT-keyphrase-extraction

**Comment by yashrooprai** [comment]
> Hey, how can I help you? You may check the code of [Pranav's github](https://github.com/pranav-ust/BERT-keyphrase-extraction) I referred for this.

Yeah I also saw that, actually I was trying to replicate your jupyter notebook and i was not able to do that exactly

**Comment by yashrooprai** [comment]
``
# Load data
test_data = data_loader.load_data('test')
# Specify the test set size
test_size = test_data['size']

data_iterator = data_loader.data_iterator(test_data, shuffle=False)

print("Starting evaluation...")
model.eval()
pred_tags = []
eval_steps = test_size//4

for _ in range(eval_steps):
  # fetch the next evaluation batch
  batch_data  = next(data_iterator)
  batch_masks = batch_data.gt(0)
  batch_output = model(batch_data, token_type_ids=None, attention_mask=batch_masks)  # shape: (batch_size, max_len, num_labels)
  
  batch_output = batch_output.detach().cpu().numpy()
  pred_tags.extend([idx2tag.get(idx) for indices in np.argmax(batch_output, axis=2) for idx in indices])
``
After executing this code I am not getting the desired pred_tags

The pred_tags arrays is containing only None valyues.

Therefore , when I run this command
``
extract = []
with open(sentences_file, 'r', encoding="utf-8") as file:
  i=0
  for line in file:
    # replace each token by its index
    tokens = line.split()
    # print(tokens)
    # sentences.append(tokens)
    temp = []
    j=0
    for token in tokens:
      if pred_tags[i][j]=='I':
        temp.append(token)
        j+=1
    if len(temp)>0:
      extract.append(temp)
    i+=1
``
I get error:

### TypeError                                 Traceback (most recent call last)
<ipython-input-16-0c44a81cad11> in <module>()
     10     j=0
     11     for token in tokens:
---> 12       if pred_tags[i][j]=='I':
     13         temp.append(token)
     14         j+=1

TypeError: 'NoneType' object is not subscriptable

**Comment by yashrooprai** [comment]
I was trying to run the code of your jupyter notebook but I was not able to get the pred_tags array, all the values in array were none so the line 

if if pred_tags[i][j]=='I'

resulted error:
TypeError: 'NoneType' object is not subscriptable
